### ISSUE TITLE
Mapping: Fn+Left key to Home and Fn+Right key to End?

### ISSUE THREAD

**Issue by rgaufman** [pqrs-org/KE-complex_modifications#525]
I used to love how I can press Fn+Left on my Apple keyboard and it will be the equivalent of pressing the "Home" key. I have now switched to a Logitech keyboard and trying to re-create this behaviour, is this possible?

**Comment by donniebreve** [comment]
I think you would just need to figure out what keycode Fn + Left outputs on the Logitech keyboard. An easy way might be to open terminal and go to preferences > keyboard, then click to remap something and enter Fn + Left and see what you get. Then you could remap that to home, or command + left.

Here is something I found:
https://support.logi.com/hc/en-001/community/posts/360036084574-Logitech-K780-Pos1-and-End-Key-on-macOS

**Comment by rgaufman** [comment]
I found out that Fn key does not output anything at all. It is only in combination that it outputs something, it is some weird firmware feature of the keyboard. I have fought with this for a long time and yes, workarounds make it sort of work but cause unexpected behaviour for other apps and then suddenly ctrl+c stops working or cmd+c stops working. I ended up getting rid of the keyboard and buying a Magic Keyboard :(

It is a shame that the great hardware is ruined by crippled firmware.
